test/unionFinde.spec.js

```diff
@@ -49,22 +49,71 @@ describe.only('#unionFind', function() {
 
 	});
 
-	it('should #count return 0 if component undefined', function() {
+	it('should #component return which component has element', function() {
+		let unionFind = new algolib.UnionFind();
+
+		unionFind.connect(1, 4);
+		unionFind.connect(4, 5);
+
+		let whichComponentHasElement = unionFind.component(1);
+		whichComponentHasElement.should.be.eql(5);
+
+	});
+	
+	it('should #count return number of existing components', function() {
```

Comment:
You have lot of `should #count return number of existing components` test 
if they test same behavior remove redundant or write better description